skrf/network.py

```diff
@@ -4129,6 +4162,55 @@ def evenodd2delta(n, z0=50, renormalize=True, doublehalf=True):
     return n_delta 
 
 
+def subnetwork(ntw, ports, offby=1):
```

Comment:
consider ntwk,  since this is used elsewere,  not a big deal

Reply:
`ntwk` instead of `nwk` that is

Reply:
sure